modAionImpl/src/org/aion/zero/impl/AionHub.java

```diff
@@ -195,6 +195,7 @@ private void initializeHub(
                 new BlockPropagationHandler(
                         1024,
                         this.blockchain,
+                        syncMgr,
```

Comment:
The `SyncMgr` should not be added as a dependency here only for the sake of gathering statistics. You can, however, pass the `SyncStats` object, or create multiple such objects and find a way to aggregate the data in a single thread for display.